src/main/java/io/federecio/dropwizard/swagger/ConfigurationHelper.java

```diff
@@ -60,7 +62,10 @@ public String getJerseyRootPath() {
                     .getJerseyRootPath();
         }
 
-        return stripUrlSlashes(rootPath);
+        if (rootPath.equals(Optional.absent()))
```

Comment:
Actually, probably should just drop the if statement and do:
`return (rootPath.or("/");`